domains/pallets/messenger/src/fees.rs

```diff
@@ -161,6 +161,17 @@ impl<T: Config> Pallet<T> {
         }
 
         if !inbox_fees.is_zero() {
+            InboxFeesOnHold::<T>::mutate(|inbox_fees_on_hold| {
+                *inbox_fees_on_hold = inbox_fees_on_hold
+                    .checked_sub(&inbox_fees)
+                    .ok_or(Error::<T>::BalanceUnderflow)?;
+
+                let imbalance = T::Currency::rescind(inbox_fees);
+                core::mem::forget(imbalance);
```

Comment:
This is the first time Subspace is forgetting an imbalance using a low-level Rust function. It might be helpful to add a comment explaining why we need to forget it (and not just drop it).

Reply:
Why not just burn the `inbox_fees` ? 
is it beacuse the burn is noted in the transfers ?

Reply:
`Currency::rescind` is kind of "burn" from the total issuance and is opposite to `Currency::issue`, we don't use `Currency::burn` because it takes an account as input and burns from the given account. And the fee will be noted as execution fee in `transfers`.

> is it beacuse the burn is noted in the transfers ?

I think this is another issue (unrelated to the change here), right now we only note fund that burn due to an account is being reaped, but if someone submit `balances::burn` extrinsic to burn fund it is not noted in the `transfers`.